### ISSUE TITLE
Run ordering

### ISSUE THREAD

**Issue by byrnedo** [polonskiy/crowdr#11]
Shouldn't the `run` command default to running containers in the order they are defined in the cfg file?

Right now it's a `tsort|tac` which runs them z-a. 

Would be great if there was some way to control the order ( or at least run them as they were defined )

**Comment by polonskiy** [comment]
Order in config doesn't matter. Run/start/stop/etc ordering is based on links. If `foo` depends on `bar`, `bar` will be started first. Check this example:

``` bash
#!/bin/bash

echo "
db image busybox
db command cat

web image busybox
web command cat
web link cache
web link db

cache image busybox
cache command cat
"
```

**Comment by byrnedo** [comment]
My use case is that I'm using consul and although there are no links
between containers, the consul container and some others need to be started
first.

**Comment by byrnedo** [comment]
I guess maybe having something like 'depends' in the config could be good
for non link based dependencies?

**Comment by marcinwaldowski** [comment]
+1

If you use user-defined networks in docker then you do not use 'link' at all. Therefore it would be nice to have something like 'depends' which work same way as 'link' in crowdr.

It looks like links are going to be deprecated ([link](https://docs.docker.com/engine/userguide/networking/dockernetworks/)):

> While links are still supported in this limited capacity, you should avoid them in preference of Docker networks. The link feature is expected to be deprecated and removed in a future release.

**Comment by polonskiy** [comment]
Fixed in 0.9.0
